apps/namadillo/src/App/Sidebars/YourStakingDistribution.tsx

```diff
@@ -66,32 +64,20 @@ export const YourStakingDistribution = ({
       >
         <div className="relative flex items-center justify-center max-w-[75%] mx-auto leading-tight">
           <AnimatePresence>
-            <span
-              className={twMerge(
-                clsx("absolute transition-opacity duration-300 opacity-100", {
-                  "opacity-0 pointer-events-none": displayedValidator,
-                })
-              )}
+            <motion.div
```

Comment:
Fixes the `key` error on console
![Screenshot 2025-03-10 at 14 11 54](https://github.com/user-attachments/assets/95fb6454-c262-42cb-aeb8-5836f571454f)